README.md

```diff
@@ -146,6 +146,9 @@ As part of the process of downloading promo codes from iTunes Connect, the user
 # Need help?
 Please submit an issue on GitHub and provide information about your setup
 
+# Code of Conduct
+Help us keep codes open and inclusive. Please read and follow our [Code of Conduct](https://github.com/fastlane/code-of-conduct).
```

Comment:
I see that @KrauseFx was suggesting that tool names be surrounded by backticks to get the monospace code presentation.

Otherwise, :ship: it!